dcpy/library/templates/dcp_mappluto.yml

```diff
@@ -4,7 +4,7 @@ dataset:
   acl: public-read
   source:
     url:
-      path: https://s-media.nyc.gov/agencies/dcp/assets/files/zip/data-tools/bytes/nyc_mappluto_{{ version }}_unclipped_shp.zip
+      path: https://s-media.nyc.gov/agencies/dcp/assets/files/zip/data-tools/bytes/nyc_mappluto_{{ version | replace(".","_") }}_unclipped_shp.zip
```

Comment:
This is so we can still have mappluto as "23v3.1" in data library but it will fetch the url with "23v3_1"